src/extensions/web-performance.ts

```diff
@@ -96,6 +100,12 @@ export class WebPerformanceObserver {
         if (this.observer) {
             return
         }
+
+        if (isLocalhost() && !this._forceAllowLocalhost) {
+            logger.log('PostHog Peformance observer not started because we are on localhost.')
```

Comment:
I'm guessing these logs aren't also ingested into PostHog?

Reply:
... Maybe, maybe not....

Reply:
But its only once on startup anyway so thats probably useful to see in the recording